tox.ini

```diff
@@ -0,0 +1,39 @@
+[tox]
+envlist = py36, py37, py38, py39
+# https://tox.readthedocs.io/en/latest/config.html#conf-requires
+# Ensure pip is new so we can install manylinux wheel
+requires = pip >= 19.0.0
+           virtualenv >= 16.0.0
+
+[testenv]
+deps =
+    pre-commit
+    future
+    numpy
+    Pillow
+    pytest
+    PyYAML
+    tables
+    pytest-rerunfailures
+    pytest-sugar
+    pytest-xdist
+    restructuredtext-lint
+    https://github.com/ome/zeroc-ice-py-github-ci/releases/download/0.1.0/zeroc_ice-3.6.5-cp36-cp36m-linux_x86_64.whl; platform_system=="Linux" and python_version=="3.6"
+    https://github.com/ome/zeroc-ice-py-github-ci/releases/download/0.1.0/zeroc_ice-3.6.5-cp37-cp37m-linux_x86_64.whl; platform_system=="Linux" and python_version=="3.7"
+    https://github.com/ome/zeroc-ice-py-github-ci/releases/download/0.1.0/zeroc_ice-3.6.5-cp38-cp38-linux_x86_64.whl; platform_system=="Linux" and python_version=="3.8"
+    https://github.com/ome/zeroc-ice-py-github-ci/releases/download/0.1.0/zeroc_ice-3.6.5-cp39-cp39-linux_x86_64.whl; platform_system=="Linux" and python_version=="3.9"
+    pywin32; platform_system=="Windows"
+    mox3
+    jinja2
```

Comment:
This list of dependencies still doesn't match the current requirements.txt and old test-requirements.txt files (some are added, some are missing). Is there a reason for the difference?

Reply:
I use the requirements file now in the tox.ini so things should be in order.